sdks/java/core/build.gradle

```diff
@@ -91,6 +91,8 @@ dependencies {
   shadow library.java.avro
   shadow library.java.snappy_java
   shadow library.java.joda_time
+  shadow library.java.json_org
+  shadow library.java.everit_json_schema
```

Comment:
I've marked is as a `provided` dependency.

Before moving forward - another possibility is to create a new extension module to hold these dependencies and this functionality. The module would be used by some of our schema transform implementations (kafka and pubsub), so it would still come back as a depenency for other Beam modules, but not for core - wdyt?